### DISCUSSION TITLE
Welcome to asynqmon Discussions!

### DISCUSSION THREAD

**Discussion by hibiken** [hibiken/asynqmon#20] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Ussik** [comment]
hey man,Import as a Library,but i encountered an error:template: pattern matches no files: `ui\build\index.html`,am I missing any steps?

**Reply by Ussik** [reply]
if RootPath:/monitoring,err is unexpected path prefix.if RootPath:/,err is template: pattern matches no files: ui\build\index.html

**Comment by IsaacDSC** [comment]
This feature is very important. I worked at a company where this technology was widely used, and we had to publish to test an event, and we could only do it with code, which always generated a new main.go to do the publishing. So why not have this on the event screen, similar to rabbitmq and other tools?

Thinking about this impact, I opened this [PR](https://github.com/hibiken/asynqmon/pull/343), let's make this project even better?
